recipe/meta.yaml

```diff
@@ -1,54 +1,55 @@
-{% set version = "0.9.4" %}
+{% set name = "terminado" %}
+{% set version = "0.13.1" %}
 
 package:
-  name: terminado
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/terminado/terminado-{{ version }}.tar.gz
-  sha256: 9a7dbcfbc2778830eeb70261bf7aa9d98a3eac8631a3afe3febeb57c12f798be
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/terminado-{{ version }}.tar.gz
+  sha256: 5b82b5c6e991f0705a76f961f43262a7fb1e55b093c16dca83f16384a7f39b7b
 
 build:
   number: 0
-  script: "{{ PYTHON }} setup.py install"
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
-    - setuptools
-    - ptyprocess  # [not win]
-    - pywinpty  # [win]
-    - tornado >=4
+    - flit
+    - pip
+    - setuptools >=40.8.0
+    - tornado
+    - wheel
   run:
     - python
-    - ptyprocess  # [not win]
-    - pywinpty  # [win]
+    - ptyprocess        # [not win]
+    - pywinpty >=1.1.0  # [win]
     - tornado >=4
 
 test:
-  imports:
-    - terminado
-
   requires:
-    - nose
+    - pip
     - posix  # [win]
-
+    - pytest
+    - pytest-cov
+  imports:
+    - terminado
   commands:
-    - nosetests terminado  # [not (osx or win)]
-    # Failing test on osx: https://github.com/conda-forge/staged-recipes/pull/313#issuecomment-208426267
-    # Failing test on Linux (due to ps1 of conda? having conda activate in bashrc? One of those two!
-    # File "/home/ray/m/conda-bld/terminado_1503494513344/_test_env_placehold_.../lib/python2.7/site-packages/terminado/tests/basic_test.py", line 84, in get_pid
-    # ..
-    # self.write_stdin("echo $$\r")
-    # (stdout, extra) = yield self.read_stdout()
-    # pid = int(stdout.split('\n')[1])
-    # ..
-    #  ValueError: invalid literal for int() with base 10: '\x1b]777;notify;Command completed;echo $$\x07\x1b]0;ray@localhost:~/m/conda-bld/terminado_1503494513344/test_tmp\x07\x1b]7;file://localhost.localdomain/home/ray/m/conda-bld/terminado_1503494513344/test_tmp\x07(root) ec'
-    #- nosetests terminado  # [not osx and not linux]
+    # see run_test.py for more
+    - pip check
+  # Downstream packages are jupyter_server and notebook
+  # Testing them to verify that they will succeed
+  downstreams:
+    - jupyter_server
+    # notebook >=6.4.6 need to be rebuild for python 3.10 support
+    #- notebook
```

Comment:
Would it be possible to add a selector `# [not py310]` or something like that, instead of disabling it completely?

Reply:
Oh, thanks! I'll rebuild it